src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  gap: 1px;
+  padding: 10px;
+  box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)

when all elements have the same `box-sizing` value, their behavior becomes more predictable